src/capitalize.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * A function equivalent to TypeScript’s built-in `Capitalize` utility type.
+ *
+ * @param string The string to capitalize.
+ * @signature
+ *    R.capitalize(string)
+ * @example
+ *    R.capitalize(`hello`) // => `Hello`
+ * @dataLast
+ * @category String
+ * @strict
+ */
+export const capitalize = <T extends string>(string: T): Capitalize<T> =>
```

Comment:
To prevent any accidental confusion, let's avoid calling the variable `string` as it's also the type name. 
```suggestion
export const capitalize = <T extends string>(str: T): Capitalize<T> =>
```